onboard/catkin_ws/src/sonar/scripts/sonar_image_processing.py

```diff
@@ -37,6 +37,39 @@ def scan_and_build_sonar_image(sonar,
     return sonar_img
 
 
+def build_color_sonar_image_from_int_array(int_array, npy_save_path=None, jpeg_save_path=None):
+    """ Build a sonar image from a list of data messages
+
+    Args:
+        data_list (List): List of data messages from either the Sonar device
+                        or from a .bin file
+        display_results (bool, optional): Whether to display the resulting
+                                        sonar image. Defaults to False.
```

Comment:
Update comment to remove args `data_list` and `display_results` and add `int_array`.